WBooks/AppDelegate.swift

```diff
@@ -10,10 +10,12 @@ import UIKit
 @main
 class AppDelegate: UIResponder, UIApplicationDelegate {
 
-
-
     func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: [UIApplication.LaunchOptionsKey: Any]?) -> Bool {
-        // Override point for customization after application launch.
+        //Customize NavigationBar
+        UINavigationBar.appearance().barTintColor = .blueHavelock
+        UINavigationBar.appearance().tintColor = .white
+        UINavigationBar.appearance().titleTextAttributes = [NSAttributedString.Key.foregroundColor: UIColor.white,
+                                                            NSAttributedString.Key.font: UIFont.KailasaBoldTitle]
```

Comment:
instead of using a comment, extract it to a method called customizeNavigationBar

Reply:
Done